build.gradle

```diff
@@ -445,6 +445,13 @@ project('segmentstore:server:host') {
     applicationDistribution.into("bin") {
         from(createAppWithGCLogging)
     }
+
+    javadoc {
```

Comment:
io.pravega.segmentstore.server.host.handler.AppendProcessor javadoc refers to AppendProcessorBuilder generated using lombok. So we need to delombok before javadoc step.